python/ray/includes/metric.pxi

```diff
@@ -0,0 +1,103 @@
+from ray.includes.metric cimport (
+    CCount,
+    CGauge,
+    CHistogram,
+    CTagKey,
+    CSum,
+    CMetric,
+)
+from libcpp.memory cimport unique_ptr
+from libcpp.string cimport string as c_string
+from libcpp.unordered_map cimport unordered_map
+from libcpp.vector cimport vector as c_vector
+
+cdef class TagKey:
+    cdef c_string __name
```

Comment:
Is there a convention for double-underscore here? Otherwise please use a single one.

Reply:
`cdef` attribute is not visible to Python. So, it's OK to use `cdef c_string name` instead.